Added privacy policy in Spanish

jenniferisasi opened the pull request:
Follows Issue #1360 , a translation of the privacy policy already published in English. Needs review from a Spanish team member to merge.

jenniferisasi committed:
Added privacy policy in Spanish

jenniferisasi committed:
Update snippets.yml

Updated the menu-about-privacy to reflect translation of our privacy policy into Spanish

arojascastro reviewed (dismissed):
muy buena traducción, he visto un esfuerzo por usar lenguaje inclusivo :)

jenniferisasi committed:
Update snippets.yml

Editing this snippet to run TRAVIS again as it says the file doesn't exist. 
es:
    title: Política de privacidad
    link: /es/politica-de-privacidad

jenniferisasi committed:
Update snippets.yml

Failed to include the / before es/ to generate the links. [hand-on-face emoji here]

acrymble commented:
@mdlincoln can you help them with the dropdown link error?

mdlincoln commented on es/politica-de-privacidad.md:
````diff
@@ -0,0 +1,62 @@
+```
````
Two changes needed:

1. remove the ``` - these don't belong around the YAML headers in a markdown file
2. Please add `original: privacy-policy` to the metadata so that the translation is properly linked

mdlincoln commented on es/politica-de-privacidad.md:
```diff
@@ -0,0 +1,62 @@
+
+## Datos de publicación
+
+Como publicación en acceso abierto comprometida con la transparencia, *Programming Historian* recopila cierta información sobre los autores, traductores, editores y revisores. Esta información se muestra de manera pública en el siguiente archivo:  https://github.com/programminghistorian/jekyll/blob/gh-pages/_data/ph_authors.yml y también se utiliza para completar varias páginas de nuestro sitio w ...
```
Please add `<>` around the URL here

jenniferisasi committed:
Update politica-de-privacidad.md

Made changes as pointed by @mdlincoln

jenniferisasi commented:
Yay! Thank you @mdlincoln for your help. I think my editor put the ``` around the yaml and I didn't see it.
